pkgs/development/tools/language-servers/verible/default.nix

```diff
@@ -1,11 +1,11 @@
-{ lib
```

Comment:
The nix convention for these is to go in the order they appear in the file, with exception of `lib` that goes first: https://ryantm.github.io/nixpkgs/contributing/coding-conventions/

Reply:
Ah, interesting, good to know. In another review, the reviewer asked me to sort these. Will have to read the documentation more throuroughly.

Anyway, fixed now.